src/test/java/org/jenkinsci/plugins/docker/workflow/DockerDSLTest.java

```diff
@@ -423,4 +423,54 @@ private static void grep(File dir, String text, String prefix, Set<String> match
             }
         });
     }
+
+    @Issue("JENKINS-26178")
+    @Test public void container_exec() {
+        story.addStep(new Statement() {
+            @Override public void evaluate() throws Throwable {
+                assumeDocker();
+                WorkflowJob p = story.j.jenkins.createProject(WorkflowJob.class, "build");
+                p.setDefinition(new CpsFlowDefinition(
+                    "node {\n" +
+                    "  writeFile file: 'Dockerfile', text: '# This is a test.\\n\\nFROM alpine\\nRUN chmod u+s /bin/busybox \\nENTRYPOINT [\\\"/bin/ping\\\" ] \\nCMD [\\\"localhost\\\"] \\n'\n" +
+                    "  def built = docker.build 'alpine-entrypoint-test'\n" +
+                    "  echo \"built ${built.id}\"\n" +
+                    "}", true));
+                WorkflowRun b = story.j.assertBuildStatusSuccess(p.scheduleBuild2(0));
+            }
+        });
+        story.addStep(new Statement() {
+            @Override public void evaluate() throws Throwable {
+                assumeDocker();
+                WorkflowJob p = story.j.jenkins.createProject(WorkflowJob.class, "prj");
+                p.setDefinition(new CpsFlowDefinition(
+                    "node {\n" +
+                    "def con = docker.image('alpine-entrypoint-test').run()\n" +
+                    "echo  con.exec('cat /etc/hosts');\n" +
+                    "def  r = con.exec('echo 42');\n" +
+                    "  echo \"the answer is ${r}\";\n" +
+                    "  con.stop();\n" +
+                    "}", true));
+                WorkflowRun b = story.j.assertBuildStatusSuccess(p.scheduleBuild2(0));
+            }
+        });
+        story.addStep(new Statement() {
+            @Override public void evaluate() throws Throwable {
+                WorkflowJob p = story.j.jenkins.getItemByFullName("prj", WorkflowJob.class);
+                WorkflowRun b = p.getLastBuild();
+                story.j.assertLogContains("ip6-localhost", b);
+                story.j.assertLogContains("the answer is 42", b);
+                DockerClient client = new DockerClient(new Launcher.LocalLauncher(StreamTaskListener.NULL), null, null);
+                String aetIID = client.inspect(new EnvVars(), "alpine-entrypoint-test", ".Id");
+                Fingerprint f = DockerFingerprints.of(aetIID);
```

Comment:
Delete all this fingerprint checking; unrelated to the purpose of this test case. I suppose just copied from some older test.